onecodex/viz/_distance.py

```diff
@@ -95,8 +97,9 @@ def plot_distance(
         ----------
         rank : {'auto', 'kingdom', 'phylum', 'class', 'order', 'family', 'genus', 'species'}, optional
             Analysis will be restricted to abundances of taxa at the specified level.
-        metric : {'braycurtis', 'manhattan', 'jaccard', 'unifrac', 'unweighted_unifrac}, optional
+        metric : {'braycurtis', 'cityblock', 'manhattan', 'jaccard', 'unifrac', 'unweighted_unifrac', 'aitchison'}, optional
             Function to use when calculating the distance between two samples.
+            Note that 'cityblock' and 'manhattan' are interchangeable metrics.
```

Comment:
I think it's clearer to use the word equivalent instead of interchangeable

Reply:
Thank you. Changed in another commit.